cranelift/interpreter/src/step.rs

```diff
@@ -630,6 +630,8 @@ where
         Opcode::Fence => unimplemented!("Fence"),
         Opcode::WideningPairwiseDotProductS => unimplemented!("WideningPairwiseDotProductS"),
         Opcode::SqmulRoundSat => unimplemented!("SqmulRoundSat"),
+        Opcode::ExtendedPairwiseAddSigned => unimplemented!("ExtendedPairwiseAddSigned"),
```

Comment:
Are these still the old opcodes?

Reply:
Glad you caught this. Yes, was going to do one more once through but may have missed this. Surprised it doesn't warn?

Reply:
Me too; @abrown I guess we're not building the CLIF interpreter by default in any CI target?

Reply:
I thought we were: there are `test interpret` filetest that should use it and @afonso360 has a fuzz target that should use it.